rpc/level0_test.go

```diff
@@ -918,11 +918,7 @@ func TestRecvBootstrapCall(t *testing.T) {
 	})
 	defer func() {
 		finishTest(t, conn, p2)
-		select {
-		case <-srvShutdown:
-		default:
-			t.Error("Bootstrap client still alive after Close returned")
-		}
+		<-srvShutdown // Hangs if bootstrap client is never shut down.
```

Comment:
I'm going to merge without. I feel like when we've hit problems before it may have been a matter of GitHub doing something weird; you're right that in a lot of these cases multiple seconds should be plenty, but we've seen it...